test/AspNet.Security.OAuth.Providers.Tests/SuperOffice/SuperOfficeTests.cs

```diff
@@ -89,6 +89,7 @@ static void ConfigureServices(IServiceCollection services)
                 services.PostConfigureAll<SuperOfficeAuthenticationOptions>((options) =>
                 {
                     options.ClientId = "gg454918d75b1b53101065c16ee51123";
+                    options.TokenValidationParameters.ValidAudience = options.ClientId;
```

Comment:
Is this just for the tests, or would a user using the provider have to set this too? If the latter, it would be nice if the provider set that for the user automatically using the internal post-configure hook.

Reply:
This is just for the tests. It works great without this setting in the MVC test client, but because the timing of PostConfigureAll it needs to be set in the test.